docs/install.md

```diff
@@ -99,12 +99,9 @@ By default Flarum's directory structure includes a `public` directory which cont
 
 However, if you wish to host Flarum in a subdirectory (like `yoursite.com/forum`), or if your host doesn't give you control over your webroot (you're stuck with something like `public_html` or `htdocs`), you can set up Flarum without the `public` directory.
 
-Simply move all the files inside the `public` directory (including `.htaccess`) into the directory you want to serve Flarum from. Then edit `.htaccess` and uncomment lines 9-14 in order to protect sensitive resources. Finally, edit both `index.php` and the `flarum` executable, and update the paths in the following lines to reflect your new directory structure:
-
-```php
-require 'vendor/autoload.php';
-```
+Simply move all the files inside the `public` directory (including `.htaccess`) into the directory you want to serve Flarum from. Then edit `.htaccess` and uncomment lines 9-14 in order to protect sensitive resources. Finally, edit the `site.php` and update the paths in the following lines to reflect your new directory structure:
 
+site.php
```

Comment:
Is this intended?

Reply:
It was not, it's been removed in the latest commit.